meta-mel/conf/local.conf.append.ls1021atwr

```diff
@@ -1,2 +1,5 @@
 # LS1021atwr supports usb-audio & simple soundcard, alsa utils are required for playback/record operations. Also we need to support vfat.
 MACHINE_FEATURES_append_ls1021atwr = " alsa vfat"
+
+# LS1021atwr install rcw into pre-built binaries.
+EXTRA_IMAGES = "rcw"
```

Comment:
This variable name is too generic, it doesn't mention archive-release or DEPLOY_IMAGES, so one can't tell, looking at this section of local.conf, that it affects archive-release.